src/ui/Helpers/ViewModelHelpers.cs

```diff
@@ -37,15 +38,19 @@ public static string GetApplicationStatus(this Course course)
         public static string GetCourseStatus(this Course course)
         {
             var result = "";
-            if (course.CourseSites.Any(s => String.Equals(s.Status, "r", StringComparison.InvariantCultureIgnoreCase)))
+            var statusRunningAndPublishedOnUcas = course.CourseSites.Any(s =>
```

Comment:
Can probably make it look samey
``` ruby
  def ucas_status
    return :running if findable? # (RY)
    return :new if new? # (N)

    :not_running # (?)
  end```

```csharp
if(statusRunningAndPublishedOnUcas) {
return Running;
} else if (course.CourseSites.Any(s => String.Equals(s.Status, "n", StringComparison.InvariantCultureIgnoreCase)))
return NewNotRunning;

return NotRunning;
```

Reply:
@defong I don't know what you mean, could you suggest an improvement to the PR?

Reply:
I meant make the ruby and csharp code syntagmatically the same looksy
```csharp
public static string GetCourseStatus(this Course course)
{
  var statusRunningAndPublishedOnUcas = course.CourseSites.Any(s =>  String.Equals(s.Status, "r", StringComparison.InvariantCultureIgnoreCase) &&
                String.Equals(s.Publish, "y", StringComparison.InvariantCultureIgnoreCase)
            );
  if(statusRunningAndPublishedOnUcas) {
    return Running;
 } else {
   if (course.CourseSites.Any(s => String.Equals(s.Status, "n",  StringComparison.InvariantCultureIgnoreCase)))
  {
   return NewNotRunning;  
  }
}
return NotRunning;
}
```